src/service/conversation-manager-service.ts

```diff
@@ -634,23 +635,60 @@ export class ConversationManagerService {
 
   public getMonitoredConversations(): { alias: string; address: string }[] {
     const monitored: { alias: string; address: string }[] = [];
+    const all = Array.from(
+      this.conversationWithContactByConversationId.values()
+    );
+
+    // active
+    all
+      .filter(({ conversation }) => conversation.status === "active")
+      .forEach(({ conversation, contact }) =>
+        monitored.push({
+          alias: conversation.myAlias,
+          address: contact.kaspaAddress,
+        })
+      );
 
-    Array.from(this.conversationWithContactByConversationId.values())
+    // pending initiated-by-us (so estimate uses self-message path)
+    all
       .filter(
-        (conversationAndContact) =>
-          conversationAndContact.conversation.status === "active"
+        ({ conversation }) =>
+          conversation.status === "sent-pending" && conversation.initiatedByMe
       )
-      .forEach((conversationAndContact) => {
-        // Monitor our own alias
+      .forEach(({ conversation, contact }) =>
         monitored.push({
-          alias: conversationAndContact.conversation.myAlias,
-          address: conversationAndContact.contact.kaspaAddress,
-        });
-      });
+          alias: conversation.myAlias,
+          address: contact.kaspaAddress,
+        })
+      );
 
     return monitored;
   }
 
+  public getConversationStatusByAddress(address: string): {
```

Comment:
Should we name it `getConversationLightInfoByAddress`? 

Are these minimal information considered "standard" in a way we will not have multiple of these different "light info" method